src/wallet/context.h

```diff
@@ -23,6 +34,9 @@ class Chain;
 struct WalletContext {
     interfaces::Chain* chain{nullptr};
     ArgsManager* args{nullptr};
+    Mutex wallets_mutex;
```

Comment:
off topic: I noticed a very nice comment https://github.com/bitcoin/bitcoin/blob/1704bbf2263f16c720604cfab4ccb775315df690/src/node/context.h#L49 in `src/node/context.h`. Would it make sense to add the comment on L36, too?

Reply:
Thanks!